algorithms/kernel/pca/pca_dense_correlation_online_impl.i

```diff
@@ -45,7 +48,11 @@ services::Status PCACorrelationKernel<online, algorithmFPType, cpu>::compute(con
     parameter->covariance->input.set(covariance::data, pData);
     parameter->covariance->parameter.outputMatrixType = covariance::correlationMatrix;
 
-    services::Status s = parameter->covariance->computeNoThrow();
+    services::Status s;
+    {
+        DAAL_ITTNOTIFY_SCOPED_TASK(compute.covariance);
```

Comment:
Don't you want to say it's a online PCA? For example, `compute.online.covariance`?